subprojects/core/src/integTest/groovy/org/gradle/JansiEndUserIntegrationTest.groovy

```diff
@@ -87,7 +90,7 @@ class JansiEndUserIntegrationTest extends AbstractIntegrationSpec {
             public class MyClass {}
         """
 
-        succeeds 'compileJava'
+        withInstallations(Jvm.current(), AvailableJavaHomes.jdk11).succeeds 'compileJava'
```

Comment:
❓ can this be done with `executor.beforeExecute` like you did below?

Reply:
It could but I don't see the advantage. Do you mean to put it in `setup` or something? I don't think that's appropriate as it doesn't apply to all the tests.

Reply:
That's what I was thinking.  It doesn't really matter which version of the JDK we're using to compile/run the tests, since we're just verifying the behavior of what happens when the test (in the build under test) runs.